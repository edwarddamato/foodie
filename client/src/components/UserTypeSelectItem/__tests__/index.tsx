import * as React from 'react';
import { UserTypeSelectItem } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const userType = {
    type: 'Chef'
  };
  const userTypeSelectItem = create(
    <UserTypeSelectItem userType={userType} />
  ).toJSON();
  expect(userTypeSelectItem).toMatchSnapshot();
});