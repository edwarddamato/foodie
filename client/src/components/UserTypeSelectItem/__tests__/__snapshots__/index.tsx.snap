// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="component_user-select-item"
>
  <button
    className="component_user-select-button"
    onClick={[Function]}
  >
    Chef
  </button>
</li>
`;
